app/src/main/cpp/DeviceDelegate.h

```diff
@@ -79,7 +79,8 @@ class DeviceDelegate {
   virtual const vrb::Matrix& GetHeadTransform() const = 0;
   virtual const vrb::Matrix& GetReorientTransform() const = 0;
   virtual void SetReorientTransform(const vrb::Matrix& aMatrix) = 0;
-  virtual void Reorient(vrb::Matrix&) = 0;
+  enum class ReorientMode { SIX_DOF, NO_ROLL };
+  virtual void Reorient(vrb::Matrix &, ReorientMode) = 0;
```

Comment:
nit: remove space. 

Shouldn't be "const vrb::Matrix&" ? If I have understood it well, the argument is only passed as input argument in either the CalculateReorientationMatrixOnHeadLock or CalculateReorientationMatrixWithThreshold and in both cases it's marked as const.

Reply:
OK